packages/sdk/README.md

```diff
@@ -1,39 +1,174 @@
-# Apillon SDK for NodeJS
+# Apillon SDK
 
 [![npm version](https://badge.fury.io/js/@apillon%2Fsdk.svg)](https://badge.fury.io/js/@apillon%2Fsdk)
 [![Twitter Follow](https://img.shields.io/twitter/follow/Apillon?style=social)](https://twitter.com/intent/follow?screen_name=Apillon)
 
-Apillon SDK is a NodeJS library written in Typescript for using Apillon Web3 services.
+Libraries and tools for interacting with your Apillon integration.
+Apillon SDK reduces the amount of work required to use Apillons REST API. It reduces the boilerplate code you need to write
+as well as compresses multi step flows into single operations.
 
 ## Requirements
 
-To be able to use Apillon CLI, you must register an account at [Apillon.io](https://app.apillon.io), create a project and generate an API key with appropriate permissions.
```

Comment:
I prefer this sentence to remain as it is more explanatory and points to registration page.